src/commands/utils/uploadArtifactOrUDF.ts

```diff
@@ -282,3 +285,27 @@ export async function promptForFunctionAndClassName(
   }
   return { functionName, className };
 }
+
+/**
+ * Submit a `CREATE FUNCTION` statement to register a UDF for the provided artifact, function and
+ * class names defined by the user, and Flink database.
+ */
+export async function executeCreateFunction(
+  artifact: FlinkArtifact,
+  userInput: {
+    functionName: string;
+    className: string;
+  },
+  database: CCloudFlinkDbKafkaCluster,
+) {
+  const ccloudResourceLoader = CCloudResourceLoader.getInstance();
+  await ccloudResourceLoader.executeFlinkStatement<{ created_at?: string }>(
```

Comment:
@Cerchie did we want to return anything from this function? I noticed we were setting the expected `{ created_at?: string }` type but never did anything with it.